src/main/java/org/opensearch/securityanalytics/threatIntel/iocscan/dao/IocMatchService.java

```diff
@@ -42,9 +54,12 @@ public class IocMatchService {
     private final Client client;
     private final ClusterService clusterService;
 
-    public IocMatchService(final Client client, final ClusterService clusterService) {
+    private final NamedXContentRegistry xContentRegistry;
+
+    public IocMatchService(final Client client, final ClusterService clusterService, final NamedXContentRegistry xContentRegistry) {
```

Comment:
plz rename to IocFindingService (and model to ioc findings

Reply:
fixed it.